packages/badge/README.md

```diff
@@ -105,41 +105,48 @@ The `<sp-badge>` can be customized with either semantic or non-semantic variants
 Non-semantic badge colors are no longer supported directly by Spectrum and Spectrum Web Components. You can mimic their delivery via the following CSS Custom Properties. These values for the `variant` attribute/property have not been marked as deprecated, but will no longer achieve the results you may have relied on in the past.
 
 ```html demo
-<div style="display: flex; gap: var(--spectrum-global-dimension-size-50);">
+<div style="display: flex; gap: var(--spectrum-spacing-75);">
     <sp-badge
         variant="seafoam"
-        style="--mod-badge-background-color-default: var(--spectrum-global-color-static-seafoam-600)"
+        static="black"
```

Comment:
Set these all to `static="black"` as the `-background-color-default` versions of all these colors are quite light.